automation/jenkins_generate_ami.sh

```diff
@@ -0,0 +1,52 @@
+#/bin/bash
+
+if [ "$deployTo" = "production" ]; then
+    S3_ACCESS_KEY=${PRODUCTION_S3_ACCESS_KEY}
+    S3_SECRET_KEY=${PRODUCTION_S3_SECRET_KEY}
+    S3_REGION=${PRODUCTION_S3_REGION:-us-east-1}
+    BALENA_PRELOAD_SSH_PUBKEY=${PRELOAD_SSH_PUBKEY_PRODUCTION}
+    BALENACLI_TOKEN=${BALENAOS_PRODUCTION_TOKEN}
+    BALENA_ENV="balena-cloud.com"
+elif [ "$deployTo" = "staging" ]; then
+    S3_ACCESS_KEY=${STAGING_S3_ACCESS_KEY}
+    S3_SECRET_KEY=${STAGING_S3_SECRET_KEY}
+    S3_REGION=${STAGING_S3_REGION:-us-east-1}
+    BALENA_PRELOAD_SSH_PUBKEY=${PRELOAD_SSH_PUBKEY_STAGING}
+    BALENACLI_TOKEN=${BALENAOS_STAGING_TOKEN}
+    BALENA_ENV="balena-staging.com"
+fi
+
+MACHINE=${JOB_NAME#yocto-}
+YOCTO_IMAGES_PATH="$WORKSPACE/build/tmp/deploy/images/${MACHINE}"
```

Comment:
If there are further artifacts you need from 'jenkins_build.sh`, copy them to the deploy folder, use them, and remove them afterwards.

Reply:
Just need the raw file, which as I understand, should be available in the `jenkins-deploy` folder as you said